src/test/java/Homework19.java

```diff
@@ -0,0 +1,56 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest {
+
+    @Test
+    public void deletePlaylist() throws InterruptedException {
+
+        String deleteNotification = "Deleted playlist lazy days";
+
+        enterEmail("christina.taylor@testpro.io");
+
+        enterPassword("jKV0uSX6z1dv");
+        submit();
+        //select lazy days
+        clickLazyDays();
+        // delete button
+
+        removePlaylist();
+
+        // ok button to conform
+        //Verify that the confirmation notification
+        // displayed has the text, "Deleted playlist {playlist name}".
+
+        // notification message
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
   
   Please, add Assert

**One test - One Assert**